docs/network/index.html

```diff
@@ -854,25 +854,62 @@ <h2 id="methods">Methods</h2>
                 <td colspan="2">Methods to get information on nodes and edges.</td>
             </tr>
             <tr class="collapsible toggle" onclick="toggleTable('methodTable','getPositions', this);">
-                <td colspan="2"><span parent="getPositions" class="right-caret" id="method_getPositions"></span> getPositions(<code><i>[Array of
-                    nodeIds]</i></code>)
+                <td colspan="2"><span parent="getPositions" class="right-caret" id="method_getPositions"></span>
+                    getPositions(<code><i>[Array of nodeIds]</i></code>, <code><i>[nodeId]</i></code>)
                 </td>
             </tr>
             <tr class="hidden" parent="getPositions">
                 <td class="midMethods">Returns: Object</td>
-                <td>Returns the x y positions in canvas space of the nodes with the supplied nodeIds as an object:
-<pre class="code">
-{
-  nodeId1: {x: xValue, y:yValue},
-  nodeId2: {x: xValue, y:yValue},
-  ...
-}
-</pre>
+                <td>
+                    Returns the x y positions in canvas space of the nodes or node with the supplied nodeIds or nodeId as an object:
+                    <pre class="code">
+// All nodes in the network.
+network.getPositions();
+>   {
+        a123: { x: 5, y: 12 },
+        b456: { x: 3, y: 4 },
+        c789: { x: 7, y: 10 }
+    }
+
+
+// Specific nodes.
+network.getPositions(['a123', 'b456']);
+>   {
+        a123: { x: 5, y: 12 },
+        b456: { x: 3, y: 4 },
+    }
+
+
+// A single node.
+network.getPositions('a123');
+>   {
+        a123: { x: 5, y: 12 }
+    }
+                    
+                    </pre>
                     Alternative inputs are a String containing a nodeId or nothing. When a String is supplied, the
```

Comment:
I have no idea why the word string was with capital s, it makes no sense to me. It should be string with lower case s.

Similar issue is with id/ID. We generally treat it as a standard word in code (nodeId, not nodeID) but it's pretty inconsistent in the docs. We should probably stick with id for the sake of consistency with the code (it's short for identifier, which is a single word, though the common spelling is different).

Reply:
Yeaaaah- I tried to look through the documentation to try to figure out a standard, but didn't really see one. Moved to string/id for the documentation that I added/altered.